.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,105 @@
+name: Benchmark
+
+# on:
+#   pull_request:
+#     types: [labeled]
+#   workflow_dispatch:
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  benchmark:
+    # if: ${{ github.event.label.name == 'run-benchmark' && github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' }}
+    name: Linux
+    runs-on: ubuntu-20.04
+    env:
+      CCACHE_BASEDIR: "${{ github.workspace }}"
```

Comment:
I think we can delete this ccache stuff since we don't use it.

Reply:
I added if-checks for these steps now. Is that fine? I have a dream that this version should work fine on scikit-image as well.